skyrl-train/examples/mini_swe_agent/preprocess_swegym.py

```diff
@@ -0,0 +1,50 @@
+"""
+Preprocess the SWEBench dataset to SkyRL format
+"""
+
+import argparse
+import os
+
+import datasets
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--output_dir", default="~/data/swe_gym_subset")
+
+    args = parser.parse_args()
+
+    args.output_dir = os.path.expanduser(args.output_dir)
+
+    data_source = "SumanthRH/SWE-Gym-Subset"
+    eval_data_source = "SumanthRH/SWE-bench_Verified"
+
+    dataset = datasets.load_dataset(data_source, "default")
+    train_dataset = dataset["train"]
```

Comment:
Maybe do `train_dataset = datasets.load_dataset(data_source, "default")["train"]` to make it consistent with `val_dataset`?